sealed-secrets.yaml

```diff
@@ -0,0 +1,118 @@
+package:
+  name: sealed-secrets
+  version: 0.28.0
+  epoch: 0
+  description: A Kubernetes controller and tool for one-way encrypted Secrets
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/bitnami-labs/sealed-secrets
+      tag: v${{package.version}}
+      expected-commit: 6b1b331a2cd3a58569ce4d819a7cabc59c0a3e50
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.36.0
+
+  - uses: go/build
+    with:
+      packages: ./cmd/controller
+      output: controller
+      ldflags: -X main.VERSION=${{package.version}}
+
+subpackages:
+  - name: ${{package.name}}-kubeseal
+    dependencies:
+      provides:
+        - kubeseal=${{package.full-version}}
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/kubeseal
+          output: kubeseal
+          ldflags: -X main.VERSION=${{package.version}}
+    test:
+      pipeline:
+        - runs: |
+            kubeseal --version | grep ${{package.version}}
+
+  - name: ${{package.name}}-kubeseal-bitnami-compat
+    description: "compat package with bitnami/sealed-secrets-kubeseal image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: sealed-secrets-kubeseal
+          version-path: 0/debian-12
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/sealed-secrets-kubeseal/bin/
+          # this is conflicting with ca-certificates-bundle
+          rm -f ${{targets.contextdir}}/etc/ssl/certs/ca-certificates.crt
+          ln -sf /usr/bin/kubeseal ${{targets.contextdir}}/opt/bitnami/sealed-secrets-kubeseal/bin/kubeseal
+          ln -sf /usr/bin/kubeseal ${{targets.contextdir}}/kubeseal
+    test:
+      pipeline:
+        - runs: |
+            stat /opt/bitnami/sealed-secrets-kubeseal/bin/kubeseal
+            stat /kubeseal
```

Comment:
Ideally, we'd install both binaries and run the same tests ran below to validate the links work as intended. We know `stat` will always pass